src/components/SignUp/ActorInfo1.tsx

```diff
@@ -1,22 +1,203 @@
+              <Form.Group>
+                <Container>
+                  <Row>
+                    <Col lg="6">
+                      <CAGLabel>Pronouns</CAGLabel>
+                      <Form.Control
+                        aria-label="pronouns"
+                        as="select"
+                        defaultValue={actorInfo1Pronouns}
+                        name="actorInfo1Pronouns"
+                        onChange={setForm}
+                      >
+                        <option value={undefined}>Choose...</option>
+                        {pronouns.map(noun => (
+                          <option value={noun}>{noun}</option>
+                        ))}
+                      </Form.Control>
+                    </Col>
+                    <Col lg="6">
+                      <CAGLabel>Other</CAGLabel>
+                      <Form.Control
+                        aria-label="pronouns"
+                        defaultValue=""
+                        disabled={actorInfo1Pronouns === 'Other' ? false : true}
+                        name="actorInfo1Pronouns"
+                        onChange={setForm}
+                      ></Form.Control>
+                    </Col>
+                  </Row>
+                </Container>
+              </Form.Group>
+              <Form.Group>
+                <CAGLabel>
+                  Do you identify as part of the LGBTQIA+ community? -{' '}
+                  <PrivateSpan>Private</PrivateSpan>
+                </CAGLabel>
+                <p>A = asexual, not ally</p>
+                <Checkbox
+                  checked={actorInfo1LGBTQ === 'Yes'}
+                  fieldType="radio"
+                  label="Yes"
+                  name="actorInfo1LGBTQ"
+                  onChange={setForm}
+                  value="Yes"
+                />
+                <Checkbox
+                  checked={actorInfo1LGBTQ === 'No'}
+                  fieldType="radio"
+                  label="No"
+                  name="actorInfo1LGBTQ"
+                  onChange={setForm}
+                  value="No"
+                />
+                <Checkbox
+                  checked={actorInfo1LGBTQ === 'I do not wish to respond'}
+                  fieldType="radio"
+                  label="I do not wish to respond"
+                  name="actorInfo1LGBTQ"
+                  onChange={setForm}
+                  value="I do not wish to respond"
+                />
+              </Form.Group>
+              <Form.Group>
+                <CAGLabel>Do you identify as any of the following:</CAGLabel>
+                <PrivacyPar>
+                  Note: This list is not exhaustive nor hyper-specific. The goal
+                  of selecting your identity is to help cast roles that call for
+                  a certain demographic. Please select as may options as you
+                  feel fits your identity. More info on our process{' '}
+                  <Link to="">here</Link>.
```

Comment:
Use `#` for the to field for now until we have somewhere to go: `<Link to="#">here</Link>`